### DISCUSSION TITLE
Import a big json file only on SSR

### DISCUSSION THREAD

**Discussion by asantanacu** [vercel/next.js#23564] [Help]
Hello
I have a problem with a json file (100kb+) I need to use only on SSR and using the import the file is always reaching the frontend as a chunk file(even with dynamic import).
I would like to know what is the right way in next js to import that file only on server side.
I am using next js v9.4.4 with a custom server.

**Comment by wwwdepot** [comment]
Load it in [`getServerSideProps`](https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering). Use https://next-code-elimination.vercel.app/ to check if it eliminates from client-side bundle.
